docs/_sass/custom/custom.scss

```
@@ -66,3 +74,33 @@ main {
     background-color: $label-error-bg;
   }
 }
+
+//Outline styles
```

I would tend to put a space here
```suggestion
// Outline styles
```